### PR TITLE
Feature/feature confirmation email

### PR DESCRIPTION
**Pull request by enrichp**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5227713&bid=17139181)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java [hunk 1]
```diff
@@ -30,24 +30,26 @@
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java:33]: This comment is not necessary

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java [hunk 2]
```diff
@@ -30,24 +30,26 @@
...
         mAuth = FirebaseAuth.getInstance();
 
         FragmentLogInBinding binding = FragmentLogInBinding.inflate(inflater, container, false);
         View view = binding.getRoot();
         binding.loginButton.setOnClickListener(v -> {
-            email = Objects.requireNonNull(binding.loginUsernameText.getText()).toString();
+            email = Objects.requireNonNull(binding.loginEmailText.getText()).toString();
             password = Objects.requireNonNull(binding.loginPasswordText.getText()).toString();
             if (!email.isEmpty() && !password.isEmpty()) {
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/LogInFragment.java]: This condition will be better if refactored for readability

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SignUpFragment.java [hunk 3]
```diff
@@ -28,50 +33,76 @@
...
     private TextInputLayout [] inputLayout;
+    private FirebaseAuth mAuth;
+    private String email;
+    private String password;
+    //private String username;
 
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
```
Comment (AlbertPG) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/SignUpFragment.java:35]: This comment is not necessary

**Review by AlbertPG — CHANGES_REQUESTED** [review]
You should consider the changes I suggested before doing the merge

**Review by AlbertPG — APPROVED** [review]
Better

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/NewPassword.java  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5227713&bid=17139181)

**Review by santidrj — APPROVED** [review]
Everything seems ok. 😃

**Pull request merged**